### ISSUE TITLE
GUI support `—ignore-leading`

### ISSUE THREAD

**Issue by ayeseeem** [pasdoc/pasdoc#144]
This option seems only available through CLI, not through GUI.

**Comment by michaliskambi** [comment]
Agreed with this and similar issues you reported ( https://github.com/pasdoc/pasdoc/issues/142 , https://github.com/pasdoc/pasdoc/issues/143 ). Indeed PasDoc GUI has some small things missing -- just not exposed in the GUI. And yes, we would like to fix it.

I'll try to deal with it at some point.

Alternatively, maybe you would like to create a PR for this? I would happily accept it. :)

**Comment by ayeseeem** [comment]
If I find any other flags that the GUI does not support, are you happy for me to create separate issues for them, or should I add them all to a single issue?

I might try to create a PR, but at the moment I am restricted to legacy tools (Delphi 6 and an old version of Lazarus) and cannot build the software. I would like to be able to help a little, because PasDoc has turned out to be really helpful to me, thank you.

**Comment by michaliskambi** [comment]
> If I find any other flags that the GUI does not support, are you happy for me to create separate issues for them, or should I add them all to a single issue?

In general I prefer to split things into more issues/PRs, as it makes them easier to review, and immediately merge the stuff that is OK (without blocking merge of X by some comment about edge-case of feature Y). So don't put too much into a single PR :)

That said, I suppose some of https://github.com/pasdoc/pasdoc/issues/144 , https://github.com/pasdoc/pasdoc/issues/143, https://github.com/pasdoc/pasdoc/issues/142 are really very simple changes to code. When it looks straightforward, I'm OK to just have one PR about them all.

Sorry for not giving a straight answer -- basically I'm OK with sometimes merging stuff into one PR. But if in doubt, separate PRs are always safer.
